kyla_stearns/Fundamentals/basic13.rb

```diff
@@ -0,0 +1,89 @@
+# Print 1-255
+p (1..255).to_a
+
+# Print odd numbers between 1-255
+(1..255).to_a.select { |elem| p elem if elem.odd? } 
+
+# Print Sum
+def print_sum
```

Comment:
This is good, but not quite the ruby way. Look into the array reduce method to clean this up.